### PR TITLE
Proofread readme [CI SKIP]

### PR DESCRIPTION
**Pull request by guangbochen**
Sync proofread fix of https://github.com/harvester/docs/pull/55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,22 +57,22 @@ During the installation, you can either choose to form a new cluster, or join th
 1. Mount the Harvester ISO disk and boot the server by selecting the `Harvester Installer`.
 ![iso-install.png](./docs/assets/iso-install.png)
 1. Choose the installation mode by either creating a new Harvester cluster, or by joining an existing one.
-1. Choose the installation device that the Harvester will be formatted to.
-1. Configure the hostname and select the network interface for the management network, by default harvester will create a bond NIC named `harvester-mgmt`, and the IP address can either be configured via DHCP or static method.
+1. Choose the installation device that Harvester will be formatted to.
+1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `harvester-mgmt`, and the IP address can either be configured via DHCP or a static assigned a static one.
```
Comment (gitlawr) [inline README.md]: `a static assigned a static one` doesn't look right

File: README.md [hunk 2]
```diff
@@ -5,14 +5,14 @@ Harvester
 [![Releases](https://img.shields.io/github/release/harvester/harvester/all.svg)](https://github.com/harvester/harvester/releases)
 [![Slack](https://img.shields.io/badge/slack-join-brightgreen)](https://slack.rancher.io/)
 
-Harvester is an open source [hyper-converged infrastructure](https://en.wikipedia.org/wiki/Hyper-converged_infrastructure) (HCI) software built on Kubernetes. It is an open alternative to using a proprietary HCI stack that incorporates the design and ethos of [Cloud Native](https://en.wikipedia.org/wiki/Cloud_native_computing). 
+Harvester is an open source [hyper-converged infrastructure](https://en.wikipedia.org/wiki/Hyper-converged_infrastructure) (HCI) software built on Kubernetes. It is an open alternative to using a proprietary HCI stack that incorporates the design and ethos of [Cloud Native](https://en.wikipedia.org/wiki/Cloud_native_computing) computing.
```
Comment (gitlawr) [inline README.md]: Include `computing` in the link text: [Cloud Native Computing]

File: README.md [hunk 3]
```diff
@@ -57,22 +57,22 @@ During the installation, you can either choose to form a new cluster, or join th
...
 1. Choose the installation mode by either creating a new Harvester cluster, or by joining an existing one.
-1. Choose the installation device that the Harvester will be formatted to.
-1. Configure the hostname and select the network interface for the management network, by default harvester will create a bond NIC named `harvester-mgmt`, and the IP address can either be configured via DHCP or static method.
+1. Choose the installation device that Harvester will be formatted to.
+1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `harvester-mgmt`, and the IP address can either be configured via DHCP or a static assigned a static one.
 ![iso-installed.png](./docs/assets/iso-nic-config.gif)
-1. (Optional) Configure the DNS servers, use commas as a delimiter.
-1. Config the Virtual IP which you can use for accessing the cluster or joining the cluster by the other nodes.
+1. (Optional) Configure the DNS servers. Use commas as a delimiter.
+1. Configure the `Virtual IP` which you can use to access the cluster or join the cluster to other nodes.
```
Comment (gitlawr) [inline README.md]: Isn't it `join other nodes to the cluster`

**Comment by guangbochen** [comment]
good catch, updated and  thanks @gitlawr

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**